pkgs/development/python-modules/rasterio/default.nix

```diff
@@ -89,13 +94,6 @@ buildPythonPackage rec {
 
   doInstallCheck = true;
```

Comment:
Does it still needed?

Reply:
Thank you. I just verified that rasterio Python tests are launched without `doInstallCheck`, so I have removed it.

Reply:
Also, explicitly set `doCheck = true;`